### PR TITLE
Updated dependencies to latest versions.

### PR DESCRIPTION
**Pull request by JonathanHenson**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: aws-common-runtime/CMakeLists.txt [hunk 1]
```diff
@@ -17,7 +17,7 @@ set(AWS_DEPS_DOWNLOAD_DIR "${AWS_DEPS_BUILD_DIR}/downloads" CACHE PATH "Dependen
 
 message("install dir ${AWS_DEPS_INSTALL_DIR}")
 set(AWS_C_COMMON_URL "https://github.com/awslabs/aws-c-common.git")
-set(AWS_C_COMMON_SHA "v0.3.11")
+set(AWS_C_COMMON_SHA "v0.3.13")
```
Comment (justinboswell) [inline aws-common-runtime/CMakeLists.txt]: There's now a v0.3.14, which includes fixes for callstack decoding tests on linux. Should be a trivial update.

File: source/http/HttpConnection.cpp [hunk 2]
```diff
@@ -39,6 +39,24 @@ namespace Aws
                 OnConnectionShutdown onConnectionShutdown;
             };
 
+            class UnManagedConnection final : public HttpClientConnection
```
Comment (justinboswell) [inline source/http/HttpConnection.cpp]: Unmanaged, perhaps?

File: tests/HttpClientConnectionManagerTest.cpp [hunk 3]
```diff
@@ -113,7 +113,7 @@ static int s_TestHttpClientConnectionManagerResourceSafety(struct aws_allocator
     connections.clear();
     for (auto &connection : connectionsCpy)
```
Comment (justinboswell) [inline tests/HttpClientConnectionManagerTest.cpp]: clearing connectionsCpy would have the same effect, wouldn't it?

Comment (JonathanHenson) [reply tests/HttpClientConnectionManagerTest.cpp]: oh.... yeah true.

**Pull request merged**